utils/playbackProfiles/helpers/codecProfiles.ts

```diff
@@ -0,0 +1,345 @@
+import {
+  CodecProfile,
+  CodecType,
+  ProfileConditionType,
+  ProfileCondition,
+  ProfileConditionValue
+} from '@jellyfin/client-axios';
+import { browserDetector } from '~/plugins/browserDetection';
+
+/**
+ * Gets the max video bitrate
+ *
+ * @returns {number | null} Returns the MaxVideoBitrate
+ */
+function getGlobalMaxVideoBitrate(): number | null {
+  let isTizenFhd = false;
+  if (browserDetector.isTizen()) {
+    try {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      // eslint-disable-next-line no-undef
+      const isTizenUhd = webapis?.productinfo?.isUdPanelSupported();
+      isTizenFhd = !isTizenUhd;
+
+      // eslint-disable-next-line no-console
+      console.debug('isTizenFhd = ' + isTizenFhd);
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error('isUdPanelSupported() error code = ' + error.code);
+    }
+  }
+
+  return browserDetector.isPs4()
+    ? 8000000
+    : browserDetector.isXbox()
+    ? 12000000
+    : browserDetector.isTizen && isTizenFhd
+    ? 20000000
```

Comment:
I think we should document this kind of hardcoded values better

Reply:
I have just copied these values from Jf-web. I'm not entirely sure where they come from. I can add a todo to investigate where they come from?

Reply:
This is exactly why I mentioned this. For now I think it's enough, nothing else we can do then.

Reply:
Cool, A comment has been added.